beacon_node/network/src/sync/block_lookups/mod.rs

```diff
@@ -786,7 +786,6 @@ impl<T: BeaconChainTypes> BlockLookups<T> {
             self.log,
             "Block component processed for lookup";
             "response_type" => ?R::response_type(),
-            "result" => ?result,
```

Comment:
does it make sense to log the `block_root` here?

Reply:
good call! added